TicTacToe/index.css

```
@@ -0,0 +1,47 @@
+button{
+  color: white;
+  background-color: black;
+  border: 0px;
+  padding: 0px;
+}
+body {
+  background-color: black;
+  text-align: center;
+  color: white;
+}
+h1{
+  font-size: 100px;
+  margin-bottom: 0;
+}
+
+#board {
+  padding-top: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.scoreboard{
+  font-size: 25px;
+  display: flex;
+  justify-content: space-around;
+  padding: 20px, 60px, 0, 60px;
```

Separado por virgulas?
```suggestion
  padding: 20px 60px 0 60px;
```